addon/models/twitch-video.js

```diff
@@ -0,0 +1,9 @@
+import Model from 'ember-data/model';
+import attr from 'ember-data/attr';
+// import { belongsTo, hasMany } from 'ember-data/relationships';
+
+export default Model.extend({
+  twitchUser: attr(),
+  twitchTeam: attr(),
+  twitchSubscription: attr()
```

Comment:
Right there with you! I've actually been building out the video model on a separate branch -- and I think this is an artifact from something I generated early on 😄. I'll take it out.